sources/packages/backend/apps/db-migrations/src/sql/EducationPrograms/Rollback-col-none-of-entrance-requirements.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    none_of_entrance_requirements DROP NOT NULL;
+
+ALTER TABLE
+    sims.education_programs
+ALTER COLUMN
+    none_of_entrance_requirements DROP DEFAULT;
```

Comment:
Shouldn't there be a rollback for the `Update` statement?

Reply:
What's the point of updating on the rollback?

Reply:
It's ok. We can leave the data values rollback.